src/inmanta/server/config.py

```diff
@@ -33,6 +33,13 @@
 db_name = Option("database", "name", "inmanta", "The name of the database on the postgresql server")
 db_username = Option("database", "username", "postgres", "The username to access the database in the PostgreSQL server")
 db_password = Option("database", "password", None, "The password that belong to the database user")
+db_connection_pool_min_size = Option(
```

Comment:
It seems that these connection parameter are never read by the Inmanta server to pass them towards the data.connect() method. Right?